examples/__testfixtures__/styled-components-to-ucl/unsupported.output.ts

```diff
@@ -0,0 +1,25 @@
+import { Box, FormControl, Input, Text } from '@rbilabs/universal-components';
+
+export const Scrollable = Box.withConfig({
+  overflowX: 'auto',
+});
+
+export const Fieldset = FormControl.withConfig({
+  padding: 0,
+})/* TODO: RN - This was a <input> tag. Verify its styled properly*/;
+
+export const Container = Box.withConfig({})/* TODO: RN - This was a <ul> tag. Verify its styled properly*/;
+
+export const Label = Text.withConfig({
+  // TODO: RN - unsupported CSS
+  // display: 'block',
+})/* TODO: RN - This was a <label> tag. This should be converted to a UL <FormControl.Label>*/;
+
+export const Group = Box.withConfig({
+  listStyle: 'none',
+})/* TODO: RN - This was a <li> tag. Verify its styled properly*/;
+
+export const Radio = Input.withConfig({
+  position: 'absolute',
+  fontFamily: Styles.fontFamily.base,
+})/* TODO: RN - This was a <input> tag. Verify its styled properly*/;
```

Comment:
I would love for this comment to come before, but I haven't had much luck. @bkasten-rbi any ideas?


I end up with something like:

```ts
export const Radio = /* TODO: RN - This was a <input> tag. Verify its styled properly*/;
Input.withConfig({
  position: 'absolute',
  fontFamily: Styles.fontFamily.base,
})
```

Reply:
i've struggled with placements of comments too.. i've basically given up because ideally we'll just delete them all anyhow